tests/upgrade_v2_script/static/fluent_bit.output.yaml

```diff
@@ -36,6 +36,9 @@ fluent-bit:
     - effect: NoSchedule
       operator: Exists
   image:
+    fluent_bit:
```

Comment:
Shouldn't this be migrated?

Reply:
But the tests on the PR passed, this means migration script tests are not run as part of PR check?

Reply:
They are https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/eb4d7a52585ec7e6fadfae55160509731075b61f/ci/tests.sh#L19-L21

Something's not right

Reply:
Yeah, it's
https://github.com/helm/charts/blob/master/stable/fluent-bit/values.yaml#L5
vs
https://github.com/fluent/helm-charts/blob/fb1f611236a3d4b4f56514bb1d05b23820f8a409/charts/fluent-bit/values.yaml#L9

Reply:
I mean - for the output the `fluent_bit` is still there, so this one should be correct (let me double check).
It's just not there for the Helm chart value.

Reply:
Hm, nope, I'm wrong 😅

Reply:
Tests fixed.